rts/Rendering/Textures/Bitmap.h

```diff
@@ -59,7 +59,7 @@ class CBitmap {
 	unsigned int CreateTexture(float aniso = 0.0f, float lodBias = 0.0f, bool mipmaps = false, uint32_t texID = 0) const;
 	unsigned int CreateMipMapTexture(float aniso = 0.0f, float lodBias = 0.0f) const { return (CreateTexture(aniso, lodBias, true)); }
 	unsigned int CreateAnisoTexture(float aniso = 0.0f, float lodBias = 0.0f) const { return (CreateTexture(aniso, lodBias, false)); }
-	unsigned int CreateDDSTexture(unsigned int texID = 0, float aniso = 0.0f, float lodBias = 0.0f, bool mipmaps = false) const;
+	unsigned int CreateDDSTexture(unsigned int texID = 0, bool mipnear = false, float aniso = 0.0f, float lodBias = 0.0f, bool mipmaps = false) const;
```

Comment:
a bit weird to insert the new param near the front, especially since there is a related one at the end already